ngi_reports/stockholm/project_summary.py

```diff
@@ -260,12 +365,14 @@ def get_accredit_info(self,keys):
             try:
                 accredit = self.proj_details['accredited_({})'.format(k)]
                 if accredit in ['Yes','No']:
-                    accredit_info[k] = "{} Validated under ISO accreditation 17025:2005".format(["[cross]","[tick]"][accredit == "Yes"])
+                    accredit_info[k] = "{} under ISO accreditation 17025:2005".format(["[cross] Not validated","[tick] Validated"][accredit == "Yes"])
                 else:
-                    accredit_info[k] = accredit
+                    accredit_info[k] = accredit.replace("N/A","Not Applicable")
```

Comment:
This sets `NA` when the field is empty, no? Struggling to remember the meeting now, but I thought that it should throw and error and skip the row when the field isn't set? As project co-ordinators should explicitly set NA? @vezzi can you remember anything about this?